cpp/CMakeLists.txt

```diff
@@ -352,12 +352,12 @@ install(TARGETS raft
 
 install(TARGETS raft_distance
         DESTINATION ${lib_dir}
-        COMPONENT raft_distance
+        COMPONENT raft
```

Comment:
So the default raft component includes the header-only versions, and requesting specific components is only for pre-built libraries? Making sure that I understand the intention here.